### PR TITLE
ran prettier

### PR DESCRIPTION
**Pull request by noahamac**
Ran prettier and followed recommendations. Many changes. Notably,

- properly typed input component events as SyntheticEvents
- removed unused imports
- standardizes all indent
- removes ts-ignores over styled @looker/components

### CODE REVIEW

**Comment by brln-looker** [comment]
are 100% of the changes in here automatic from prettier?

File: src/components/Fields.tsx [hunk 1]
```diff
@@ -36,46 +36,49 @@ import {
...
   &:last-child {
     border-bottom: none;
   }
-`;
+`
 
 // Sticky Table Header
-// @ts-ignore
-export const StickyHeader = styled(TableHeaderCell)`
+export const StickyHeader = styled(TableHeaderCell as any)`
```
Comment (noahamac) [inline src/components/Fields.tsx:60]: There are a few things Prettier wouldn't do, but recommended I do. Removing ts-ignore and adding "as any" was something I did manually

File: src/components/PanelFields.tsx [hunk 2]
```diff
@@ -213,7 +240,9 @@ export const PanelFields: React.FC<{
               hideSearchIcon
               placeholder="Filter fields in this Explore"
               mt="medium"
-              onChange={e => setSearch(e.currentTarget.value)}
+              onChange={(e: SyntheticEvent) => {
```
Comment (noahamac) [inline src/components/PanelFields.tsx:243]: Properly casting input events as Synthetic events was another

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "test": "jest",
     "build": "webpack --mode=production",
     "lint": "eslint 'src/components/*.ts{,x}'",
-    "lint-fix": "eslint '**/*.ts{,x}' --fix"
+    "lint-fix": "eslint 'src/components/*.ts{,x}' --fix"
```
Comment (noahamac) [inline package.json:10]: Had to change this manually

**Comment by noahamac** [comment]
Everything else was done by prettier with `--fix`

**Review by brln-looker — APPROVED** [review]

**Pull request merged**